tests/backup/backup_delete_test.go

```diff
@@ -467,3 +468,238 @@ var _ = Describe("{DeleteIncrementalBackupsAndRecreateNew}", func() {
+			for _, provider := range providers {
+				DeleteBucket(provider, localBucketNameMap[provider])
+				log.Infof("Sleeping for default 10 minutes for next backup sync service to be triggered")
+				time.Sleep(10 * time.Minute)
+			}
+		})
+
+		Step("Verify the backups are in CloudBackupMissing state after bucket deletion", func() {
+			log.InfoD("Verify the backups are in CloudBackupMissing state after bucket deletion")
+			var wg sync.WaitGroup
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, backupName := range backupNames {
+				wg.Add(1)
+				go func(backupName string) {
+					defer GinkgoRecover()
+					defer wg.Done()
+					bkpUid, err := Inst().Backup.GetBackupUID(ctx, backupName, orgID)
+					log.FailOnError(err, "Fetching backup uid")
+					backupInspectRequest := &api.BackupInspectRequest{
+						Name:  backupName,
+						Uid:   bkpUid,
+						OrgId: orgID,
+					}
+					requiredStatus := api.BackupInfo_StatusInfo_CloudBackupMissing
+					backupCloudBackupMissingCheckFunc := func() (interface{}, bool, error) {
+						resp, err := Inst().Backup.InspectBackup(ctx, backupInspectRequest)
+						if err != nil {
+							return "", false, err
+						}
+						actual := resp.GetBackup().GetStatus().Status
+						if actual == requiredStatus {
+							return "", false, nil
+						}
+						return "", true, fmt.Errorf("backup status for [%s] expected was [%v] but got [%s]", backupName, requiredStatus, actual)
+					}
+					_, err = DoRetryWithTimeoutWithGinkgoRecover(backupCloudBackupMissingCheckFunc, 20*time.Minute, 30*time.Second)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verfiying backup %s is in CloudBackup missing state", backupName))
+				}(backupName)
+			}
+			wg.Wait()
+		})
+
+		Step("Resume the existing backup schedules", func() {
+			log.InfoD("Resume the existing backup schedules")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, scheduleName := range scheduleNames {
+				scheduleUid, err := GetScheduleUID(scheduleName, orgID, ctx)
+				log.FailOnError(err, "Error while getting schedule uid %v", scheduleName)
+				err = resumeBackupSchedule(scheduleName, scheduleUid, periodicSchedulePolicyName, periodicSchedulePolicyUid, orgID, ctx)
+				dash.VerifySafely(err, nil, fmt.Sprintf("Verification of resuming backup schedule - %s", scheduleName))
+			}
+			log.Infof("Waiting 5 minute for another schedule backup to trigger")
+			time.Sleep(5 * time.Minute)
+		})
+
+		Step("Get the latest schedule backup and verify the backup status", func() {
+			log.InfoD("Get the latest schedule backup and verify the backup status")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, scheduleName := range scheduleNames {
+				latestScheduleBkpName, err := GetLatestScheduleBackupName(ctx, scheduleName, orgID)
```

Comment:
How are you making sure that by this time a backup is created? Don't we have to wait for a new backup to be created and reach success state?

Reply:
we have script induced sleep time of 15 minutes before we check for next schedule backup. So it's sure that another set of schedule backup is triggered. Also we are checking the backup success in subsequent line.